week2/32-dlina-posliedovatiel-nosti/task.py

```diff
@@ -0,0 +1,11 @@
+def main():
+    result = 0
+    currentNumber = int(input())
+    while currentNumber != 0:
```

Comment:
try to use  while current_number := int(input()) != 0:, to avoid repetition of "currentNumber = int(input())" expression